code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -113,7 +113,7 @@
 	return 1//Humans can use guns and such
 
 /mob/living/carbon/human/InCritical()
-	return (health <= config.health_threshold_crit && stat == UNCONSCIOUS)
+	return (health <= config.health_threshold_crit)

```

Comment:
you've removed this config check in a bunch of places, are you supporting it or not?